### ISSUE TITLE
In start screen Modern UI, NVDA doesn't announce the selected candidate during a search.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2777]
**Reported by aleskis on 2012-11-03 16:35**
1. In Windows 8, Press Windows to access start screen modern UI.
2. Try to search a program like Contacts in typing its name.
3. Narrator announces the names of selected element. NVDA doesn't.

**Comment by bhavyashah** [comment]
I use Classic Shell, a software that has modified my Start menu, thus making me unable to personally test the current standing of the reported issue. Could someone else having access to a non-modified Windows 8.1 Start menu please test and revert with results?

**Comment by Adriani90** [comment]
We didn't get any updates on this issue since many years, and support for Windows 8 ended completely in 2016 while Windows 8.1 ended completely in 2023. I am closing this issue as abandoned since I don't expect any futher updates regarding this windows version.
